### ISSUE TITLE
Does Gophish supports Gmail API (Send emails using Gmail’s API)?

### ISSUE THREAD

**Issue by LeiHsiFu** [gophish/gophish#2851]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 
I use Docker image latest
[latest](https://hub.docker.com/layers/gophish/gophish/latest/images/sha256-8a57cd171999dcd1fa33f346d6898440913ae14be114ffc850cc678354672d29?context=explore)

Brief description of the issue: 
About configuring how Gophish send the email, I want to use a free Gmail account. wondering if Gophish supports Gmail API, if yes, please kindly let me know the setups please

What are you expecting to see happen? :
For "Sending Profile", does it support using [Gmail API](https://kinsta.com/blog/gmail-smtp-server/) to send the email?

What are you seeing happen?

<img width="621" alt="SCR-20230503-kbi" src="https://user-images.githubusercontent.com/16631259/235846907-891a1e36-6aac-4b43-a757-2a6c99edc9df.png">

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - 
 * Step 2 -

**Comment by atheriqbal** [comment]
Yes, it does support. If Multi factor is enabled for Gmail account, you need to use the application specific password. 
Check link [https://support.google.com/mail/answer/185833?hl=en](url)

**Comment by glennzw** [comment]
@LeiHsiFu We don't directly support sending via the GMail API (unless you use the SMTP server as @atheriqbal mentioned) but I did write this decoupled tool to allow it:

https://github.com/glennzw/GoBridge

---
_GoBridge is an SMTP to GMail API service. It will allow you to host an SMTP server that will accept emails and directly INSERT them into users' inboxes, bypassing any filtering. This is achieved by using the GMail API; and specifically the users.messages.insert endpoint._
